games-action/polymc/polymc-1.4.2.ebuild

```diff
@@ -33,9 +33,9 @@ fi
 
 # Apache-2.0 for MultiMC (PolyMC is forked from it)
 # GPL-3 for PolyMC
-# LGPL-3 for libnbtplusplus
+# LGPL-3+ for libnbtplusplus
 # See the rest of PolyMC's libraries at https://github.com/PolyMC/PolyMC/tree/develop/libraries
-LICENSE="Apache-2.0 Boost-1.0 BSD BSD-2 GPL-2+ GPL-3 LGPL-3 OFL-1.1 MIT"
+LICENSE="Apache-2.0 BSD BSD-2 GPL-2+ GPL-3 ISC LGPL-2.1-with-linking-exception LGPL-2.1+ LGPL-3+ MIT"
```

Comment:
The only think LGPL with linking exception is quazip, which you aren't distributing

Reply:
it is still included as a subproject used in PolyMC's compilation, so i guess i'd make sense to let the user know that?

From the Gentoo wiki:
> If the package sources include additional files that are neither installed nor used at build time, their license should not be listed. 

since QuaZip is used at build-time, we should display it's license as well, if I understand correctly?

Reply:
or, i guess i'm just thinking about the version with bundled quazip xc
maybe i shouldn't mess with licenses after just waking up, i'll change it when i get home.